src/main/java/org/entando/kubernetes/service/digitalexchange/BundleUtilities.java

```diff
@@ -281,4 +283,22 @@ public static BundleType extractBundleTypeFromBundle(EntandoDeBundle entandoDeBu
                         .valueOf(bundleTypeEntry.getValue().toUpperCase().replace("-", "_")))
```

Comment:
probably be better to assign an explicit value to the enum

Reply:
updated